src/webui/api/authcontroller.cpp

```diff
@@ -74,27 +70,29 @@ void AuthController::loginAction()
         LogMsg(tr("WebAPI login success. IP: %1").arg(clientAddr));
     }
     else {
-        increaseFailedAttempts();
+        if (Preferences::instance()->getWebUIMaxAuthFailCount() > 0)
```

Comment:
0 has special meaning. Shouldn't you indicate it in UI? E.g. "Ban client after consecutive failures: Never".

Reply:
OK, done.
https://github.com/qbittorrent/qBittorrent/compare/7e1022c9439d9a6263c1ad5c930f13fd4cef0d4b..2d530d7bdd42c80144cc636c1d9137ae010de90e